tasks/read_file.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+main() {
+  cat "$PT_path"
+}
+
+outfile=$(mktemp)
+main "$@" >"$outfile" 2>&1
+exit_code=$?
+
+cat <<EOS
+  {
+    "content": $(python -c "import json; print json.dumps(open('$outfile','r').read())")
```

Comment:
fwiw, python isn't available by default on rhel 8... I'm actually confused what was wrong with actually using `cat` here.

Reply:
That's good to know. Hmm.

Python is needed/used because the result returned needs to be JSON, and special characters like newlines need to be JSON encoded to be returned.

Looks like `python` isn't available by default on Ubuntu 18.04 either.

I think the workaround for now is gonna have to be shipping a Ruby implementation of this task. At the moment we only use it after Puppet is installed, so that should work ok.

Reply:
FWIW `/usr/bin/python3` is installed on my Platform9 Ubuntu VMs.
Yes, ruby ...